onnxruntime/core/providers/cpu/tensor/utils.h

```diff
@@ -265,6 +300,27 @@ struct SliceIteratorBase {
     return out_bytes;
   }
 
+  void* CopyMaxDataInOneStep(void* output) {
```

Comment:
regarding the name `CopyMaxDataInOneStep`, what does "step" mean here? how about something like `CopyContiguousInnermostAxes`? though then it feels odd to be calling `CopyInnermostAxisNonSolitaryInnerStep` from it.